gull_tracking/gull_tracking.ipynb

```diff
@@ -37,13 +31,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "hv.extension('bokeh')\n",
-    "\n",
-    "map_tiles  = EsriImagery().opts(alpha=1.0, width=800, height=800, bgcolor='black')\n",
-    "points     = hv.Points(df, ['longitude', 'latitude'])\n",
-    "rasterized = shade(rasterize(points, x_sampling=1, y_sampling=1, width=800, height=800), cmap=fire)\n",
+    "df = pd.read_csv('./data/HG_OOSTENDE-gps-2018.csv', usecols=['location-long', 'location-lat'])\n",
+    "df.columns = ['longitude', 'latitude']\n",
+    "df.loc[:,'longitude'], df.loc[:,'latitude'] = en_ll(df.longitude, df.latitude)\n",
```

Comment:
Please add some text to explain this step, similar to the following:

The CSV file contains latitude and longitude information in degrees. Here we will want to overlay this data onto geographic mapping tiles that use the [Web Mercator projection](https://en.wikipedia.org/wiki/Web_Mercator_projection), so we use a utility from HoloViews to project the spherical coordinates into the tiles' coordinate system to allow them to be overlaid.

Reply:
You might also consider removing the `df.columns = ['longitude', 'latitude']` step and simply keeping the original names; I don't think it's any clearer to rename them and it adds an extra line to the file.